lms/djangoapps/ccx/tests/utils.py

```diff
@@ -77,6 +85,17 @@ def setUp(self):
 
         # Create instructor account
         self.coach = AdminFactory.create()
+
+        # adding staff to master course.
+        self.staff = UserFactory()
```

Comment:
@noisecapella ``TestAdminAccessCoachDashboard`` is not available in mit instance of edx.